docs-2.0/nebula-dashboard-ent/4.cluster-operator/6.settings.md

```diff
@@ -0,0 +1,21 @@
+# Other settings
+
+This topic will describe other settings in Nebula Dashboard.
+
+- Information: It shows cluster name, creates time, and creator.
+
+- Unbind: Unbind a cluster and remove its information from the platform, doing nothing to the cluster's services and data.
```

Comment:
, doing nothing to the cluster's services and data. --too oral
Data of Nebula Graph services will be retained.

Reply:
plz refer to the English page.